README.adoc

```diff
@@ -0,0 +1,38 @@
+= Fabric8-auth-cli
+
+`fabric8-auth-cli` provides a tool to retrieve an access token and optionally a refresh token from the CLI, by performing authentication on the preview or production platform and using the user's credentials which can be provided either from the terminal prompt, or using a keyring interface (using the https://github.com/zalando/go-keyring[go-keyring] library).
+
+== How to install
+
+```
+git clone https://github.com/fabric8-services/fabric8-auth-cli.git
+make install
+```
+
+== How to use it
+
+```
+osio login --help
+```
+
+On macOS, you can register a password for a service/account pair which can be used with the `osio` command so you don't need to input your OpenShift.io account's password each time:
```

Comment:
you may want to add that Linux *should* be supported following this instructions : 

https://github.com/zalando/go-keyring#linux

Reply:
done in 066e2fe